test/show.jl

```diff
@@ -0,0 +1,73 @@
+using HypothesisTests, Base.Test
+
+# based on power_divergence.jl tests
+d = [[762,484] [327,239] [468,477]]
+m = PowerDivergenceTest(d)
+
+@test sprint(show, m) == """Pearson's Chi-square Test
+-------------------------
+Population details:
+    parameter of interest:   Multinomial Probabilities
+    value under h_0:         [0.255231, 0.19671, 0.11594, 0.0893561, 0.193574, 0.14919]
+    point estimate:          [0.276387, 0.175553, 0.118607, 0.0866884, 0.16975, 0.173014]
+    95% confidence interval: Tuple{Float64,Float64}[(0.2579, 0.2955), (0.1571, 0.1947), (0.1001, 0.1378), (0.0682, 0.1058), (0.1513, 0.1889), (0.1545, 0.1922)]
```

Comment:
While you're at it, if you want to make more changes in another PR, it looks like this output could be improved by turning it into a table similar to `StatsBase.CoefTable` (probably using that type), with one parameter per row, and one column for the value under h_0, on for the point estimate, and two for the CI. It would also make sense to stop printing `Tuple{Float64, Float64}`. That would only apply to the situation where there are several parameters to estimate of course.